python/README.md

```diff
@@ -15,7 +15,7 @@ The images below include variations for only the core packages in the [Intel® D
 
 ## Build from Source
```

Comment:
after discussion with Alex Rybkin, it seems like build from source option is not relevant for our target users. So, we probably should remove this section if there are no objections